spec/_includes/foo.md

```diff
@@ -0,0 +1 @@
+foo
```

Comment:
This file looks like it was accidentally added?

Reply:
I added to test the _includes directory.  I forgot to remove.